hapi/hedera-protobufs/services/state/roster/roster.proto

```diff
@@ -45,10 +45,16 @@ message Roster {
  *
  * Each roster entry SHALL encapsulate the elements required
  * to manage node participation in the Threshold Signature Scheme (TSS).<br/>
- * All fields except tss_encryption_key are REQUIRED.
+ * All fields are REQUIRED.
  */
 message RosterEntry {
 
+    // The tssEncryptionKey field has been reserved because it is no longer
+    // required to be stored in the Roster. The public portion of this key will
+    // continue to be stored in the Roster state, but will, however, be stored
+    // just as raw bytes.
+    reserved 4;
```

Comment:
Is it too late to simply remove the field and re-number the following elements?   I know we're wanting to ship block streams as clean as possible without deprecated content in day one.

Reply:
Makes sense now. Have updated